### DISCUSSION TITLE
Is the iModel service down?

### DISCUSSION THREAD

**Discussion by kardoka** [iTwin/itwinjs-core#4588] [Q&A] [answered]
Greetings,

Is the iModel service down, or is it just me? I cannot seem to be able to load up any iModels at the moment.

```
{
    "errorType": "403",
    "errorName": "Could not find Project",
    "errorMessage": "We can't find the project that you are looking for or it does not exist."
}`

**Comment by grigasp** [comment, marked as answer]
Hi. The [403 error](https://en.wikipedia.org/wiki/HTTP_403) generally means you don't have access to the requested resource. Please make sure you're properly authenticated and have access to the requested project.

**Reply by kardoka** [reply]
Thank you for the answer. I do have access to the resource. The error seemed to have disappeared once I cleaned all of the cookies and tried logging in again.
